specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/applicationWhitelistings.json

```diff
@@ -359,9 +359,9 @@
         "VersionAndAboveSignature"
       ]
     },
-    "AppWhitelistingIssue": {
+    "AdaptiveApplicationControlIssue": {
```

Comment:
These name changes in your schemas will create breaking changes in all generated SDKs.  Please revert.